src/components/PaginationFooter/ManageStaffPagination.jsx

```diff
@@ -9,7 +9,7 @@ import {
   PaginationContainer,
 } from '@ajna/pagination';
 
-import formatStaffData from '../../utils/manageStaffUtils';
+import formatStaffData from '../../utils/ManageStaffUtils';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../../utils/ManageStaffUtils'.